tsl/src/compression/compression.c

```diff
@@ -1061,6 +1060,14 @@ decompress_chunk(Oid in_table, Oid out_table)
 			.decompressed_datums = palloc(sizeof(Datum) * out_desc->natts),
```

Comment:
Don't we need to init this field as well in such case? Was thinking about something like `calloc` analogue for memory contexts, if such thing exists

Reply:
we could do palloc0 here but you are not supposed to look at datum when is_null is true